Tests/TestUtilities/DummyExeHelper.cs

```diff
@@ -128,7 +129,10 @@ private static void CompileAssembly(string code, string outputFilePath)
 
             if (result.Errors.Count > 0)
             {
-                Console.WriteLine(result.Output.ToString());
+                foreach(string item in result.Output)
+                {
+                    Console.WriteLine(item);
+                }
```

Comment:
Output.ToString() didn't dump the log output. It's necessary to loop through output and dump each line explicitly to see the compiler errors.